packages/patternfly-4/react-core/src/components/Text/Text.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+
+export enum TextVariants {
+  h1 = 'h1',
+  h2 = 'h2',
+  h3 = 'h3',
+  h4 = 'h4',
+  h5 = 'h5',
+  h6 = 'h6',
+  p = 'p',
+  a = 'a',
+  small = 'small',
+  blockquote ='blockquote',
+  pre ='pre'
+}
+
+export interface TextProps extends React.HTMLProps<HTMLElement> {
+  /** The text component */
+  component?: TextVariants;
```

Comment:
I know it's a pain, but for the sake of docgen (and allowing the consumer to just pass as string, which we are aiming to support), can you make this `TextVariants | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p' | 'a' | 'small' | 'blockquote' | 'pre'`?

Reply:
I think this is fine as is.  We want consumers to use the enumerations for consistency and type checking.